canonical/rebac-admin
# WD-7098 - Fix mockServiceWorker.js

## Done

- Fix serving mockServiceWorker.js in the local demo.

## QA

- Run `yarn clean && yarn install`.
- Run `yarn start`.
- Visit the localhost url and check that the mocks start.
- Visit the ip and check that the mocks start.
-  Visit the [demo url](https://rebac-admin-29.demos.haus/) on this PR and check that the mocks start.

## Details

- https://warthogs.atlassian.net/browse/WD-7098
